src/libstd/lib.rs

```diff
@@ -94,6 +94,16 @@
 //! compiler - but they are documented here the same). Like the prelude, the
 //! standard macros are imported by default into all crates.
 //!
+//! # Contributing changes to the documentation
+//!
+//! Check out the rust contribution guidelines [here](https://github.com/rust-lang/rust/blob/master/CONTRIBUTING.md).
+//! The source for this documentation can be found on [Github](https://github.com/rust-lang/rust/tree/master/src/libstd).
```

Comment:
A bunch of the source lives in `libcore`, `liballoc`, `libcollections`, should probably include pointers there as well.